specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/applicationWhitelistings.json

```diff
@@ -168,6 +168,46 @@
             }
           }
         }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete an application control VM/server group": {
+            "$ref": "./examples/ApplicationWhitelistings/DeleteApplicationWhitelistings_example.json"
+          }
+        },
+        "tags": [
+          "applicationWhitelistings"
+        ],
+        "description": "Delete an application control VM/server group",
+        "operationId": "AdaptiveApplicationControls_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/AscLocation"
+          },
+          {
+            "$ref": "#/parameters/GroupName"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "404": {
+            "description": "NotFound"
+          },
```

Comment:
Should it be 204 when we are trying to delete an item that does not exist?
Randomly picked example: https://github.com/Azure/azure-rest-api-specs/blob/114c018edda56641cf2615125c3dd8a397f875d3/specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json#L143

Reply:
@ArcturusZhang  done